packages/core/src/html/NodeProcessor.js

```diff
@@ -1,6 +1,7 @@
 const path = require('path');
 
 const cheerio = require('cheerio');
+const lodashHas = require('lodash/has');
```

Comment:
We already have this require as `_.has` several lines below, let's just use that to remove duplication